### PR TITLE
Remove jquery

### PR DESCRIPTION
**Pull request by vladshcherbin**
Related to [issue](https://github.com/kadirahq/meteor-dochead/issues/16)

### CODE REVIEW

File: lib/both.js [hunk 1]
```diff
@@ -54,25 +54,46 @@ DocHead = {
     }
   },
   _buildTag(metaInfo, type) {
-    let props = "";
-    for (let key in metaInfo) {
-      props += `${key}="${metaInfo[key]}" `;
+    let meta;
+    if (Meteor.isClient) {
+      meta = document.createElement(type);
```
Comment (arunoda) [inline lib/both.js]: You don't need to create a html node here. You can simply append string to the DOM using `insertAdjacentHTML`. See: http://stackoverflow.com/a/7327125

With that, you don't need to touch this code.

File: lib/both.js [hunk 2]
```diff
@@ -54,25 +54,46 @@ DocHead = {
...
-    props += 'dochead="1"';
-    var meta = `<${type} ${props}/>`;
     return meta;
   },
   _addLdJsonScript(stringifiedObject) {
-    const scriptTag = `<script type="application/ld+json" dochead="1">${stringifiedObject}</script>`;
+    let scriptTag;
     if (Meteor.isClient) {
-      $('head').append(scriptTag);
+      scriptTag = document.createElement('script');
```
Comment (arunoda) [inline lib/both.js]: Use `insertAdjacentHTML`

**Comment by vladshcherbin** [comment]
@arunoda thanks, updated

**Comment by arunoda** [comment]
Published as `v1.3.0`

**Pull request merged**